File: CS/10_Conversion/ConvertPPSToPPTX.cs
using System;
using System.Windows.Forms;
using Spire.Presentation;

namespace ConvertPPSToPPTX
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create an instance of presentation document
            Presentation ppt = new Presentation();
            //Load file
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\Conversion.pps");

            //Save the PPS document to PPTX file format
            string result = "ConvertPPSToPPTX.pptx";
            ppt.SaveToFile(result, FileFormat.Pptx2013);
            //Launch and view the resulted PPTX file
            PresentationDocViewer(result);
        }
	
	    private void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/10_Conversion/ConvertPPSToPPTX.vb
Imports Spire.Presentation

Namespace ConvertPPSToPPTX
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create an instance of presentation document
			Dim ppt As New Presentation()
			'Load file
			ppt.LoadFromFile("..\..\..\..\..\..\Data\Conversion.pps")

			'Save the PPS document to PPTX file format
			Dim result As String = "ConvertPPSToPPTX.pptx"
			ppt.SaveToFile(result, FileFormat.Pptx2013)
			'Launch and view the resulted PPTX file
			PresentationDocViewer(result)
		End Sub

		Private Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace